Orange/preprocess/preprocess.py

```diff
@@ -34,12 +34,20 @@ def __call__(self, data):
 
 
 class Continuize(Preprocess):
-    (Indicators, FirstAsBase, FrequentAsBase, Remove, RemoveMultinomial,
-     ReportError, AsOrdinal, AsNormalizedOrdinal, Leave) = Enum(
-         "Continuize",
-         "Indicators, FirstAsBase, FrequentAsBase,"
-         "Remove, RemoveMultinomial, ReportError, AsOrdinal,"
-         "AsNormalizedOrdinal, Leave")
+    class ContinuizeDV(Enum):
+        Indicators = "Indicators"
+        FirstAsBase = "FirstAsBase"
+        FrequentAsBase = "FrequentAsBase"
+        Remove = "Remove"
+        RemoveMultinomial = "RemoveMultinomial"
+        ReportError = "ReportError"
+        AsOrdinal = "AsOrdinal"
+        AsNormalizedOrdinal = "AsNormalizedOrdinal"
+        Leave = "Leave"
```

Comment:
I don't oppose strings at all. They are simple and robust. Sklearn is all strings in params, and nobody is complaining. But sklearn checks their user input for invalid values. Could we check the passed argument is valid? E.g.:
```py
assert multinomial_treatment in self._MultinomialTreatment.__dict__
```